hello-world-attest-fn-call/Makefile

```diff
@@ -0,0 +1,36 @@
+default: run
+
+tmp:
+	@mkdir -p tmp
+
+check:
+	@command -v docker >/dev/null 2>&1 || { echo >&2 "docker not found in PATH. Aborting."; exit 1; }
+	@command -v bky-as >/dev/null 2>&1 || { echo >&2 "bky-as not found in PATH. Aborting."; exit 1; }
+	@command -v jq >/dev/null 2>&1 || { echo >&2 "jq not found in PATH. Aborting."; exit 1; }
+	
+tmp/x.wasm: main.go $(wildcard as/*.go) | tmp check
+	@echo "Building WASM module..."
+	@docker run --rm \
+        -v .:/src \
+        -w /src \
+        tinygo/tinygo:0.31.2 \
+        tinygo build -o tmp/x.wasm -target=wasi main.go
+
+build: tmp/x.wasm
+
+FUNCTION := "hello-world"
+
+run: build | check
+	@echo "Running WASM module..."
+	@cat $(FUNCTION)-call.json \
+	| bky-as attest-fn-call > tmp/out.json
+	@echo "Result:"
+	@jq -r '.function_calls[0].claims.output | @base64d | fromjson' tmp/out.json
+	@LOG=$$(jq -r '.function_calls[0].logs' tmp/out.json); \
+	if [ "$$LOG" != "null" ]; \
```

Comment:
See comments in the readme for more, but consider updating so that there are two targets to reduce the complexity of the makefile.

Reply:
OK, I'd like to keep this DRY, but I added the hello-world and hello-error targets to simplify the usage in the readme